packages/trader/src/AppV2/Utils/positions-utils.ts

```diff
@@ -55,3 +55,21 @@ export const getProfit = (
 export const getTotalPositionsProfit = (positions: (TPortfolioPosition | TClosedPosition)[]) => {
     return positions.reduce((sum, { contract_info }) => sum + Number(getProfit(contract_info)), 0);
 };
+
+export const getTabIndexFromURL = () => {
+    const searchParams = new URLSearchParams(window.location.search);
+    if (searchParams.toString()) {
+        const current_opened_tab = [...searchParams.values()];
+        return current_opened_tab[0] === 'open' ? 0 : 1;
+    }
+    return 0;
+};
+
+export const setPositionURLParams = (tab_name: string) => {
+    const searchParams = new URLSearchParams(window.location.search);
+    searchParams.set('tab_name', tab_name);
```

Comment:
I was trying to name query `tab`, but found out, that in `client-store` (in `async init(login_new_user)`) there is a function with hardcoded query params, which are being hided from url. And `type` was among them, so I that is why I used `tab_name`